index.js

```diff
@@ -15,14 +16,15 @@ const ARGS = [
     { name: 'create', type: String },
     { name: 'dest', type: String },
     { name: 'source', type: String },
+    { name: 'pysource', type: String },
     { name: 'code', type: String }
 ];
 
 const args = CLA(ARGS);
 
 let usage = 'Usage:\n';
     usage += '    j18n  --build  path/to/translations  [--dest path]\n';
-    usage += '\n    j18n  --update path/to/translations  --source glob\n';
+    usage += '\n    j18n  --update path/to/translations  --source glob  --pysource pyglob\n';
     usage += '\n    j18n  --create path/to/translations  --code lang';
```

Comment:
what do you think about using hard-coded paths, rather than having the user specify them?

the reality is that this is always going to be a submodule in the `jamovi` repo, so these could simply look in `../server`, `../client` .. in fact i wonder if this doesn't actually belong in the `jamovi` repo itself?